assets/app/scripts/controllers/create/nextSteps.js

```diff
@@ -0,0 +1,79 @@
+"use strict";
+/* jshint unused: false */
+
+/**
+ * @ngdoc function
+ * @name openshiftConsole.controller:CreateFinishedController
+ * @description
+ * # CreateFinishedController
+ * Controller of the openshiftConsole
+ */
+angular.module("openshiftConsole")
+  .controller("CreateFinishedController", function($scope, $http, $routeParams, DataService, $q, $location, TaskList, $parse, Navigate, $filter, imageObjectRefFilter, failureObjectNameFilter) {
```

Comment:
name the controller consistent with the name of the file, any reason to not call this NextStepsController